app/models/concerns/score.rb

```diff
@@ -4,7 +4,7 @@ module Score
   included do
     include Mongoid::Timestamps
 
-    belongs_to :user, foreign_key: :uid
+    belongs_to :user, foreign_key: :uid, optional: true
```

Comment:
Interesting... I have mixed feelings about this change. I like the idea of leaving this behavior unchanged, but I like also the idea of requiring an associated object by default. In general though the idea of requiring an associated object to be present in a document-oriented DB doesn't feel right.

Reply:
I just did it to satisfy some test failures, so it is up to you guys as to how you want to proceed.

Reply:
Ok, let's keep the old behavior.